lib/models/mixins/consumable.js

```diff
@@ -3,12 +3,18 @@ module.exports = provider => superclass => class extends superclass {
     return [
       ...super.IN_PAYLOAD,
       'consumed',
+      'consumedCount',
     ];
   }
 
   consume() {
     provider.emit('token.consumed', this);
-    return this.adapter.consume(this.jti);
+    if (!this.consumed) {
+      return this.adapter.consume(this.jti);
+    } else {
+      this.consumedCount = 1 + (this.consumedCount || 1);
+      return this.save();
```

Comment:
Why is the `consumedCount` necessary?

Reply:
consumed count is for tracking when expensify fails to rotate a token. if we can see consumed count > 1 we can go back to them and say "you failed to rotate tokens properly"